src/traces/box/defaults.js

```diff
@@ -29,6 +29,9 @@ function supplyDefaults(traceIn, traceOut, defaultColor, layout) {
     coerce('whiskerwidth');
     coerce('boxmean');
 
+    var notched = coerce('notched');
+    if(notched) coerce('notchwidth');
```

Comment:
I wonder if we can get away with just a `notchwidth` similar to how whiskers are set (with `whiskerwidth`)?

Reply:
Are you suggesting to remove `if(notched)`?
Btw. this line was modified after this comment: https://github.com/krassowski/plotly.js/pull/1#discussion_r164287086

Reply:
> Are you suggesting to remove if(notched)?

Yes, that's what I'm suggesting. Waiting on a concensus.

Reply:
I think @etpinard was suggesting removing the `notched` attribute altogether and just defaulting to `notchwidth=0` as no notch. I thought about this but would prefer the existing way so that we can pick a nice default `notchwidth` rather than everyone making up random values. One thing we *could* do though is make the default for notched be `true` if `traceIn.notchwidth !== undefined`. That way if you do want a custom width you don’t also need to provide `notched`.

Reply:
>  was suggesting removing the notched attribute altogether and just defaulting to notchwidth=0 as no notch

yes, this is what I meant. Sorry for the confusion.

Reply:
> I thought about this but would prefer the existing way so that we can pick a nice default notchwidth rather than everyone making up random values

Good point here. I wonder if we should do the same for whiskers in v2? That is, adding a `showwhiskers` boolean and making the `whiskerswidth` default _smarter_?

Reply:
> I wonder if we should do the same for whiskers in v2?

My argument doesn't really apply to whiskers - though I'm not sure I get what you mean by *smarter* default. If we're happy with the default `whiskerwidth` staying at `0.5` then people can use `0` (a very non-random number 😄 ) to disable whiskers and any other number for a custom width.

Reply:
@krassowski in talking with @etpinard offline, we agreed that despite the correspondence with `whiskerwidth` the Matlab convention makes more sense, to us anyway - so that's 0=no notch, 0.5=meet in the middle, with a default of 0.25.

And re: my comment:
> One thing we *could* do though is make the default for notched be `true` if `traceIn.notchwidth !== undefined`.

That's just removing `attributes.notched.dflt` and changing its coerce line to:
`var notched = coerce('notched', traceIn.notchwidth !== undefined);`

You'll also want to merge in the latest master so tests pass, then I think we're ready to go!